packages/query-bar/src/components/QueryUrl.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState, useRef } from 'react';
+import clsx from 'clsx';
+import {
+  Button,
+  IconButton,
+  Tooltip,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  withStyles,
+} from '@material-ui/core';
+import CopyIcon from '../assets/CopyIcon.svg';
+
+const QueryUrl = ({
+  classes,
+  filterItems,
+  localFind = {},
+  rootPath,
+}) => {
+  const [display, setDisplay] = useState(false);
+  const toggleDisplay = () => setDisplay((prevDisplay) => !prevDisplay);
+
+  const [open, setOpen] = useState(false);
+  const toggleOpen = () => setOpen((prevOpen) => !prevOpen);
+
+  const pathFilterParams = filterItems.reduce((acc, item) => {
+    const { datafield, items = [] } = item;
+    acc[datafield] = items;
+    return acc;
+  }, {});
+
+  const query = JSON.stringify({
+    ...pathFilterParams,
+    ...localFind,
+  });
+  const url = encodeURI(rootPath.concat(query));
```

Comment:
There is an issue when the query includes a forward slash "/". For example, if you select "B Lymphoblastic Leukemia/Lymphoma, NOS" from the "Primary Diagnosis" facet, then the query URL will fail. 

Suggested change:
`const url = rootPath.concat(encodeURIComponent(query));`

This will also have to be matched in the CDS FE PR to use decodeURIComponent instead in order for this to work.
Ex.
`const query = decodeURIComponent(match.params.filterQuery || '');`